code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -9,7 +9,7 @@
 #define SIMPLE_MOB_TISSUE_DAMAGE_STAMINA_MULTIPLIER 1.7 //Fights with simple mobs are usually more sustained, so apply a bit less
 #define BLUNT_TISSUE_DAMAGE_STAMINA_MULTIPLIER 1.8 //Brute that isnt sharp, knocks the wind outta you real good
 #define OTHER_TISSUE_DAMAGE_STAMINA_MULTIPLIER 1.5 //Burns, sharp implements
-#define PROJECTILE_TISSUE_DAMAGE_STAMINA_MULTIPLIER 1.6
+#define PROJECTILE_TISSUE_DAMAGE_STAMINA_MULTIPLIER 0
```

Comment:
1 is a much better starting place then absolute 0, as 1 is much easier to tweak up and down from, whereas 0 leaves no room for feedback other then, "Lethals do a lot less damage now"

Reply:
I agree, after thinking about it.
My first reflex was that only the bullets supposed to deal it, should actually deal it it.
But that is most likely too extreme of a starting place.